### PR TITLE
Eng 11264

### PR DESCRIPTION
**Pull request by manjujames**

### CODE REVIEW

**Comment by manjujames** [comment]
New control response message with structured message types for different response types.

File: src/protobuf/dragent.proto [hunk 1]
```diff
@@ -69,6 +69,46 @@ message Response {
...
+message ConnectResponse {
+    optional fixed64 catalogCRC = 1;
+    optional string catalogSignature = 2;
+    optional int32 protocolVersion = 3;
+    repeated ClusterInfo clusterInfo = 4;
+}
+
+message ClusterInfo {
+    required int32 clusterId = 1;
+    required fixed64 clusterPersistedId = 2;
```
Comment (nshi) [inline src/protobuf/dragent.proto]: Is this the creation instance ID?

Comment (manjujames) [inline src/protobuf/dragent.proto]: Yes. I am fine with using whatever the agreed terminology is. Let me know.

Comment (nshi) [inline src/protobuf/dragent.proto]: `clusterCreationId` is probably okay.

**Pull request merged**